fastcrypto/src/tests/bls12377_tests.rs

```diff
@@ -433,31 +433,20 @@ fn test_sk_zeroization_on_drop() {
         bytes_ptr = &sk.as_ref()[0] as *const u8;
 
         // Assert the exact location in SecretKey is stored as private key bytes.
-        unsafe {
-            for (i, &byte) in sk_bytes
-                .iter()
-                .enumerate()
-                .take(CELO_BLS_PRIVATE_KEY_LENGTH)
-            {
-                assert_eq!(*ptr.add(i + 41), byte);
-            }
-        }
```

Comment:
i think this assert is fine to keep? just line 458 needs to be modified from "assert each equal" to "assert not all equal"

Reply:
That's true. I just rewrote the loop in the same style as the faulty one because it seemed cleaner to me. But it is a redundant modification.